examples/cf-stackdriver-example/Gopkg.toml

```diff
@@ -0,0 +1,4 @@
+
+[prune]
+  go-tests = true
+  unused-packages = true
```

Comment:
I think it's worth validating that you're in compliance with all of the dependencies' licenses by excluding certain sources.